### ISSUE TITLE
Octopuses are offset from glass when glass is inside a bootstrap container

### ISSUE THREAD

**Issue by dylankainth** [PerceptronV/glass-octopus#1]
![image](https://user-images.githubusercontent.com/50585244/164988433-87f6165c-9209-4346-8e3f-1a38a9fc1a23.png)

Octopuses are offset from glass when glass is inside a bootstrap container.
Find code below.
```
<div class="col-lg-5 mx-auto p-3 pt-md-5">
    <div id="inject"></div>
    [...]
</div>
```

**Comment by dylankainth** [comment]
[full code](https://drive.google.com/file/d/1Xva02wzTbI1Q7eLni0BWjjDVibbje95a/view?usp=sharing)

**Comment by PerceptronV** [comment]
Fixed by v2.0 beta commit
